### ISSUE TITLE
scyllaadm user files are with root:root permissions needs to be updated

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla-machine-image#125]
file aws/ami/files/scylla_install_ami

```
    run('ln -sf /home/scyllaadm {}'.format(homedir))
    for skel in glob.glob('/etc/skel/.*'):
        if not skel.endswith('.bash_profile'):
            shutil.copy(skel, '/home/scyllaadm')
< here >
    run('useradd -o -u 1000 -g scyllaadm -s /bin/bash -d /home/scyllaadm centos')
    if distro == 'ubuntu':
        run('useradd -o -u 1000 -g scyllaadm -s /bin/bash -d /home/scyllaadm ubuntu')
```

We are copying files and we need to change the permissions `chown -R scyllaadm:scyllaadm /home/scyllaadm`

**Comment by shoshan** [comment]
@penberg Please port fix to 4.5  
 chown skel files to scyllaadm:scyllaadm
Latest commit 0b684ba

in 4.5 test failed with 
'mkdir: cannot create directory ‘/home/scyllaadm/.cassandra/’: Permission denied'

**Comment by penberg** [comment]
@shoshan Backported to 4.5
